packages/cli-kit/src/private/node/ui/refresh-documentation.ts

```diff
@@ -0,0 +1,77 @@
+import {examples} from './examples.js'
+import {unstyled} from '../../../public/node/output.js'
+import {JSDocTag, Project} from 'ts-morph'
+import isEqual from 'lodash/isEqual.js'
+
+async function refreshDocumentation(): Promise<void> {
+  const project = new Project({
+    tsConfigFilePath: 'tsconfig.json',
```

Comment:
I'd use an absolute path here for two reasons:
- It's easier to reason about. It's hard to tell by looking at this line if it will use the `cli-kit`'s `tsconfig.json` or the root one.
- It makes the script work regardless of which working directory you are (assuming they are building the absolute path internally from the working directory).